mail_print/views/mail_message_report.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <template id="mail_message">
+            <t t-call="report.html_container">
+                <t t-foreach="docs" t-as="o">
+                    <t t-call="report.internal_layout">
+                        <div class="page">
+                            <style>
+                                .table1 {width: 50%}
+                                .table2 {width: 100%}
+                                .font {font-family: Helvetica, sans-serif;font-size: 10pt}
```

Comment:
setting a fixed size here but not touching it for everything else can lead to weird effects depending on the browser settings. What is you rationale to pick a font and size in the first place?